### ISSUE TITLE
Tips for Training Qwen-VL as a Classification Model Using finetune_cls.sh

### ISSUE THREAD

**Issue by Cleverus-AI** [2U1/Qwen2-VL-Finetune#174]
I’d like to share a few notes that helped me run finetune_cls.sh successfully.

### 1. Prepare data.json with decoded string labels

```
[
  {
    "id": "...",
    "image": "...",
    "prompt": "...",
    "label": "target_event_1",
  },

  {
    "id": "...",
    "image": "...",
    "prompt": "...",
    "label": "no_event",
  }
  ...
]
```

### 2. Update CLASS_2_ID in cls_dataset.py
- Path: src/train/train_cls.py
- Map your decoded string labels to integer IDs:

```
CLASS_2_ID = {
    "no_event": 0,
    "target_event_1": 1,
}
```

### 3. Comment out data_args.sampler_pos_ratio
- path : src/train/train_cls.py

```
local_rank = training_args.local_rank
compute_dtype = (torch.float16 if training_args.fp16 else (torch.bfloat16 if training_args.bf16 else torch.float32))
data_args.compute_dtype = compute_dtype
# data_args.sampler_pos_ratio = training_args.sampler_pos_ratio
```

After applying these changes, I was able to run the script successfully ✨
Thanks again for this great application! 🙏

**Comment by TUDelftHao** [comment]
> I’d like to share a few notes that helped me run finetune_cls.sh successfully.
> 
> ### 1. Prepare data.json with decoded string labels
> ```
> [
>   {
>     "id": "...",
>     "image": "...",
>     "prompt": "...",
>     "label": "target_event_1",
>   },
> 
>   {
>     "id": "...",
>     "image": "...",
>     "prompt": "...",
>     "label": "no_event",
>   }
>   ...
> ]
> ```
> 
> ### 2. Update CLASS_2_ID in cls_dataset.py
> * Path: src/train/train_cls.py
> * Map your decoded string labels to integer IDs:
> 
> ```
> CLASS_2_ID = {
>     "no_event": 0,
>     "target_event_1": 1,
> }
> ```
> 
> ### 3. Comment out data_args.sampler_pos_ratio
> * path : src/train/train_cls.py
> 
> ```
> local_rank = training_args.local_rank
> compute_dtype = (torch.float16 if training_args.fp16 else (torch.bfloat16 if training_args.bf16 else torch.float32))
> data_args.compute_dtype = compute_dtype
> # data_args.sampler_pos_ratio = training_args.sampler_pos_ratio
> ```
> 
> After applying these changes, I was able to run the script successfully ✨ Thanks again for this great application! 🙏

请问启动脚本里--freeze_llm True \--freeze_vision_tower False \--freeze_merger False \ 以及学习率之类的需要修改吗？脚本默认的学习率对应globel batch size为多少呢？
